src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1158,6 +1158,18 @@ elementFormDefault="qualified">
     <xs:element name="MinimumVisualStudioVersion" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
     <xs:element name="AdditionalFileItemNames" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
     <xs:element name="AllowUnsafeBlocks" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
+    <xs:element name="AnalysisMode" type="msb:StringPropertyType" substitutionGroup="msb:Property">
```

Comment:
Thanks @drewnoakes. While you are adding code analysis related properties, can you also add 3 other boolean properties mentioned here: https://docs.microsoft.com/visualstudio/code-quality/disable-code-analysis?view=vs-2019#net-framework-projects? They apply to all .NET projects.

```xml
<RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
<RunAnalyzersDuringLiveAnalysis>false</RunAnalyzersDuringLiveAnalysis>
<RunAnalyzers>false</RunAnalyzers>
```

Reply:
@mavasani done.

Reply:
Thanks @drewnoakes!